### PR TITLE
SWC-5773

### PR DESCRIPTION
**Pull request by jay-hodgson**
Note that I will need to pull in a new SRC to avoid duplicate definitions of the box-shadow.

### CODE REVIEW

File: src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java [hunk 1]
```diff
@@ -10,6 +10,7 @@
 import org.gwtbootstrap3.client.ui.base.TextBoxBase;
 import org.gwtbootstrap3.client.ui.constants.ValidationState;
 import org.gwtbootstrap3.client.ui.html.Div;
+import org.gwtbootstrap3.client.ui.html.Paragraph;
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java:13]: Use a paragraph when rendering the Bio section.  https://sagebionetworks.jira.com/browse/SWC-5773

File: src/main/webapp/Portal.html [hunk 2]
```diff
@@ -134,7 +134,8 @@
 			head.load(cdnEndpoint + "generated/react-bootstrap.min.js");
 			head.load(cdnEndpoint + "generated/sanitize-html.min.js");
 			head.load(cdnEndpoint + "generated/synapse-react-client.production.min.js");
-			head.load(cdnEndpoint + "generated/synapse-react-client.production.styles.css");
+			// SRC scss should be compiled in SWC project
+			// head.load(cdnEndpoint + "generated/synapse-react-client.production.styles.css");
```
Comment (jay-hodgson) [inline src/main/webapp/Portal.html:138]: swc.scss file contains the following code to pull in and build the SRC scss inside the SWC project:

// Load the SCSS build after the minified build so styles with overrides take higher priority
@use 'node_modules/synapse-react-client/dist/style/main.scss' with (
  $primary-action-color: swc.$synapse-blue,
  $secondary-action-color: swc.$synapse-green,
  $bootstrap-4-danger-bg: swc.$synapse-red-error,
  $bootstrap-4-danger-text-color: white
);

File: src/main/webapp/sass/_core.scss [hunk 3]
```diff
@@ -2,7 +2,7 @@
 Synapse color scheme
 ****************/
 $synapse-green: #469285;
-$synapse-blue: #2E4761;
+$synapse-blue: #395979;
```
Comment (jay-hodgson) [inline src/main/webapp/sass/_core.scss:5]: Small change from design (in general they want a darker blue for the primary color).

File: src/main/webapp/sass/_core.scss [hunk 4]
```diff
@@ -2617,6 +2617,9 @@ td.datePickerMonth {
 .whitespace-nowrap {
 	white-space: nowrap;
 }
+.whitespace-prewrap {
+	white-space: pre-wrap; //preserve, but wrap when necessary
```
Comment (jay-hodgson) [inline src/main/webapp/sass/_core.scss:2621]: Use this in the User Profile bio paragraph

File: src/main/webapp/sass/_core.scss [hunk 5]
```diff
@@ -3450,6 +3453,12 @@ div.border {
 }
 
 #body .profileContainer {
+	.profileLinksContainer {
```
Comment (jay-hodgson) [inline src/main/webapp/sass/_core.scss:3456]: Moved some style from SRC to SWC.  This styles the container that is used to hold the new User Links component

**Review by xschildw — APPROVED** [review]

**Pull request merged**